### PR TITLE
Adding reporting issues section

### PR DESCRIPTION
**Pull request by adborbas**

### CODE REVIEW

**Review by mateherber — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,3 +39,9 @@ In the meantime, please feel free to share your feature ideas on [Discuss](https
 This section is intended to describe the coding conventions applied for the Step. Please refer to it when contributing code to the repository.
 
 Our single guide currently is to apply [Effective Go](https://golang.org/doc/effective_go.html), which has a great number of tips on writing clear, idiomatic Go code.
```
Comment (BanyikAnna) [inline CONTRIBUTING.md]: ...on writing clear and idiomatic Go.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,3 +39,9 @@ In the meantime, please feel free to share your feature ideas on [Discuss](https
...
+
+## Reporting issues
+
+Please refer to our issue template chooser if something is not behaving as expected while using the step. The template chooser should guide you to the appropriate forum to raise your question so we can assist you the fastest.
```
Comment (BanyikAnna) [inline CONTRIBUTING.md]: step >Step.

Comment (BanyikAnna) [inline CONTRIBUTING.md]: The template picker guides you to the appropriate forum where you can ask your question and we can quickly assist you. (I just like template picker better but it's up to you folks. Also I deleted "should" cause I think this picker either guides them to the forum or it fails and won't guide them anywhere, correct? :)

Comment (adborbas) [reply CONTRIBUTING.md]: Github uses the chooser terminology, but I agree with you, picker sounds better. Let's go with it and see if the users get it.

**Review by mateherber — APPROVED** [review]

**Pull request merged**